snuba/datasets/dataset_schemas.py

```diff
@@ -24,7 +25,7 @@ def __init__(
     def get_read_schema(self) -> Schema:
         return self.__read_schema
 
-    def get_write_schema(self) -> Schema:
+    def get_write_schema(self) -> WritableTableSchema:
         return self.__write_schema
 
     def __get_unique_schemas(self) -> List[Schema]:
```

Comment:
I didn't notice this before. Shouldn't this just be a set? Or are these not hashable?